atlassian_jwt_auth/contrib/tests/test_requests.py

```diff
@@ -5,7 +5,7 @@
 
 import atlassian_jwt_auth
 from atlassian_jwt_auth.tests import utils
-from atlassian_jwt_auth.contrib.requests import JWTAuth, create_jwt_auth
+from atlassian_jwt_auth.contrib.requests_auth import JWTAuth, create_jwt_auth
```

Comment:
This would be breaking api change is there any reason you cannot use `from atlassian_jwt_auth.contrib.requests as requests_auth` ?